charts/cloudnative-pg/values.yaml

```diff
@@ -145,10 +145,15 @@ monitoring:
     namespace: ""
     # -- The name of the ConfigMap containing the dashboard.
     configMapName: "cnpg-grafana-dashboard"
-    # -- Label that ConfigMaps should have to be loaded as dashboards.
+    # -- Label that ConfigMaps should have to be loaded as dashboards.  DEPRECATED: Use labels instead.
     sidecarLabel: "grafana_dashboard"
-    # -- Label value that ConfigMaps should have to be loaded as dashboards.
+    # -- Label value that ConfigMaps should have to be loaded as dashboards.  DEPRECATED: Use labels instead.
     sidecarLabelValue: "1"
+    # -- Labels that ConfigMaps should have to get configured in Grafana.
+    labels:
+      grafana_dashboard: "1"
+    # -- Annotations that ConfigMaps can have to get configured in Grafana.
+    annotations: {}
 
 # Default monitoring queries
 monitoringQueriesConfigMap:
```

Comment:
Is this going to be passed to the dependency because we set monitoring as alias for it? 🤔

Reply:
Precisely. Because there is no way to access the template from the parent chart. It needs to be generated by the subchart.